joss/paper.md

```diff
@@ -51,20 +51,20 @@ _tfaip_ resolves the following different aspects in an elegant and robust way.
 During research, a scenario is usually configured via a command line interface (CLI) which usually must be implemented by the user itself.
 _tfaip_ already provides a powerful CLI which is dynamically created upon runtime by parsing a nested dataclass hierarchy.
 To add a new parameter or even a new set of sub parameters, a user simply has to add a new field to the respective dataclass.
-Compared to other approached where all possible parameters are simultaneously available in the CLI, the dynamic approach of _tfaip_ only shows and parsed the available arguments.
-This prevents users from making mistakes by setting parameters without effect for the current configuration (e.g, setting the factor for an Adam optimizer even though RMSprop was selected).
+Compared to other approached where all possible parameters are simultaneously available in the CLI, the dynamic approach of _tfaip_ only shows the available arguments.
+This prevents users from making mistakes by setting parameters without effect for the current configuration (e.g., setting the factor for an Adam optimizer even though RMSprop was selected).
 The default CLI of _tfaip_ provides commands to adapt various hyper-parameters such as the learning rate and its schedule, the optimizer, logging, debugging, profiling, or early stopping.
 Furthermore, each component of the scenario can itself be fully customized which allows, for example, to dynamically configure the network architecture, e.g., by inserting layers or changing their parameters.
 This feature helps researchers to set up various experiments for example to optimize hyper-parameters or test novel ideas.
 
-In comparison to other frameworks such as Tensorflow, _tfaip_ requires users to implement their scenarios in object-oriented programming and encourages them to annotate their code with types.
-This is particularly sensible if the scenarios and thus their code basis becomes larger since it leads to a clean, structured, modularized, and readable code preventing bad code style and facilitates maintenance.
+In comparison to other frameworks such as Tensorflow, _tfaip_ requires users to implement their scenarios in an object-oriented way and encourages them to annotate their code with types.
+This is particularly advantageous for larger, more complex scenarios where the codebase grows accordingly. Using _tfaip_ leads to a clean, structured, modularized, and readable code limiting poor coding styles and facilitating maintenance.
 In practice, each scenario is created by implementing predefined interfaces (e.g., loss-function or the graph construction).
 
 During research and development, a tedious step is data preparation which often comprises the conversion of data into the format required by the framework.
-The Tensorflow-backed of _tfaip_ allows integrating Python code in the data pipeline which is however not run (truly) in parallel by multiple processes and results quite often in a bottleneck.
```

Comment:
I think the "of" is mandatory here, since `tfaip` is no Tensorflow backend, but vice-versa.

Reply:
Yes, Tensorflow-backed _tfaip_ means '_tfaip_ supported/backed by Tensorflow'.

Leaving 'of' in this sentence doesn't make sense.

Reply:
You are absolutely right, my mistake, I missed the missing an "n":
* Either `Tensorflow-backed _tfaip_` or
* `Tensorflow-backeNd of _tfaip_`.

Sorry.